vuepress/docs/next/docs/getting-started/concepts-overview.md

```diff
@@ -208,58 +200,57 @@ In this table you can see what are the default routes that are exposed for each
 <td align="left"><p>Entando component manager</p></td>
 </tr>
 <tr class="even">
-<td align="left"><p><code>/&lt;plugin-ingressPath&gt;</code></p></td>
+<td align="left"><p><code>/plugin-ingressPath</code></p></td>
 <td align="left"><p>Entando plugin linked to the app</p></td>
 </tr>
 <tr class="odd">
 <td align="left"><p>Entando plugin ingress</p></td>
-<td align="left"><p><code>/&lt;plugin-ingressPath&gt;</code></p></td>
+<td align="left"><p><code>/plugin-ingressPath</code></p></td>
 <td align="left"><p>Entando plugin</p></td>
 </tr>
 </tbody>
 </table>
 
-**Note**: The Entando plugin `ingressPath` variable is defined in the Entando Plugin custom resource under the `spec` element. The plugin ingress path is also used to expose the plugin under the Entando App domain
+**Note**: The Entando plugin `ingressPath` variable is defined in the Entando Plugin custom resource under the `spec` element. The plugin ingress path is also used to expose the plugin under the Entando App domain.
 
 ### Exposing Plugins in the EntandoApp Domain
 The process of exposing an Entando Plugin under the same domain (ingress) of the Entando App is made possible from the creation of an `EntandoAppPluginLink` custom resource and the corresponding controller.
 
-Once the link between the Entando App and the Entando Plugin is created, the controller reads the specification of such link and automatically creates the HTTP routes in the Entando App so that the plugin is exposed on the same domain as the App builder, App engine and Component manager.
+Once the link between the Entando App and the Entando Plugin is created, the controller reads the specification of the link and automatically creates the HTTP routes in the Entando App so that the plugin is exposed on the same domain as the App builder, App engine and Component Manager.
 
-This allows the Microfrontend developers that need to access the plugin to not worry about CORS policy or knowing what is the full path where the plugin is exposed. The plugin is referencable using relative urls.
+This allows the micro frontend developers, who need access to the plugin, to disregard CORS policy and the full path where the plugin is exposed. The plugin can be referenced using relative URLs.
 
-### How to check Ingresses in my Cluster
+### How to Check Ingresses in my Cluster
 
 #### Using the Openshift Dashboard
 
-On the Openshift dashboard, Ingresses are not exposed directly as pods and deployments. The dashbaord provides direct access to the Ingress Routes (or simply routes) under the `Applications > Routes` menu.
+On the Openshift dashboard, Ingresses are not exposed directly as pods and deployments. The dashbaord provides direct access to the Ingress Routes (or simply routes) under the `Applications` → `Routes` menu.
```

Comment:
Typo:  dashbaord -> dashboard

Reply:
should Ingresses be capitalized in this case? it has not been on other pages. maybe the usage is different.

Reply:
I think it's old language, maybe referring to OpenShift 3.11. I can see Ingresses and Routes in the OpenShift 4.8 UI.  

Let's hold off on rewriting this page and add it to the post-7.0 list. I just started looking at it and it needs more attention.

Reply:
i wish i had seen this many minutes ago

Reply:
Sorry...

Reply:
i just thought i was supposed to let the future tenses slide, not everything! i was wondering how jinah could have missed so many capitalizations. that should have been a huge hint... *gets more coffee*